snap/snapcraft.yaml

```diff
@@ -24,40 +25,47 @@ layout:
 apps:
   hotsos:
     command: bin/hotsos
+
 parts:
-  hotsos-cli:
-    plugin: dump
-    source: .
-    override-build: |
-      rm -rf tests
-      git rev-parse --short HEAD > repo-info
-      snapcraftctl build
-    organize:
-      defs: etc/hotsos/defs
-    build-packages: [git]
   hotsos:
-    plugin: python
+    plugin: dump
     source: .
-    stage-packages: [coreutils, bsdmainutils, systemd, dpkg, python3-yaml,
-                     python3-simplejson, python3-click]
-    python-packages:
-      - cryptography==3.4.8
-      - wheel
-      - pip
-      - setuptools
-    requirements:
-      - requirements.txt
+    build-packages:
+      - git
+      - python3-pip
+      - python3-venv
+      - rsync
+    stage-packages:
+      - bsdmainutils
+      - coreutils
+      - dpkg
+      - libc6
+      - libpython3.8-minimal
+      - libpython3.8-stdlib
+      - python3.8-minimal
+      - systemd
     build-environment:
       - "CRYPTOGRAPHY_DONT_BUILD_RUST": "1"
+    override-build: |
+      git rev-parse --short HEAD > repo-info
+      python3 -m venv ${SNAPCRAFT_PART_INSTALL}
+      ${SNAPCRAFT_PART_INSTALL}/bin/pip install -r requirements.txt
+      ${SNAPCRAFT_PART_INSTALL}/bin/python3 setup.py install
+      rsync -av defs ${SNAPCRAFT_PART_INSTALL}/etc/hotsos/
```

Comment:
this bit (rsync) doesn't seem relevant to the crypto issue, is it needed?

Reply:
Fixed in latest iteration